src/libraries/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -5,10 +5,16 @@
     <StringResourcesPath>..\..\System.Private.Xml\src\Resources\Strings.resx</StringResourcesPath>
     <StringResourcesName>FxResources.$(AssemblyName.Replace('-', '_')).SR</StringResourcesName>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
I don't think we shoudl change this one.  It's not a normal library.  Its a tool.

Reply:
I'm fine with not changing it but then do you want me to skip testing on this one? The reason why I did it for this too is that now with the new test I added it'll fail for any package that depends on netstandard for framework, so that includes this too.

Reply:
Probably, we likely aren't testing it the right way.  It's supposed to behave like a tool (although it's pretty questionable as it has a props file with a DotNetCliToolReference in it).  cc @ViktorHofer

Reply:
I suspect we should just avoid changing this one and opt it out of testing.

Reply:
Done.